Source/Diagnostics/ReducedDiags/Costs.H

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2019-2020 Yinjian Zhao
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_DIAGNOSTICS_REDUCEDDIAGS_COSTS_H_
+#define WARPX_DIAGNOSTICS_REDUCEDDIAGS_COSTS_H_
+
+#include "WarpX.H"
+#include "ReducedDiags.H"
+
+/**
+ *  This class mainly contains a function that update the
+ *  costs (used in load balance) for writing to output.
+ */
+class Costs : public ReducedDiags
```

Comment:
I feel like it's better to give the reduced diagnostics a more specific name, like "LoadBalanceCosts" rather than just "Costs".

Reply:
Updated to `LoadBalanceCosts` .